### PR TITLE
Added grid size example

### PR DESCRIPTION
**Pull request by arcadiaEvgeniyCheremukhin**

### CODE REVIEW

File: src/examples/grid-size/example-grid-size.component.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<button (click)="fillLarge()" class="buttonMarginStyle">Fill 100%</button>
```
Comment (klumba12) [inline src/examples/grid-size/example-grid-size.component.html:1]: please remove class from the buttons

File: src/examples/grid-size/example-grid-size.component.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+	fillMedium() {
+		this.setWidthAndHeight('60%');
+	}
+
+	fillExact() {
+		this.setWidthAndHeight('500px');
+	}
+
+	setWidthAndHeight(size) {
```
Comment (klumba12) [inline src/examples/grid-size/example-grid-size.component.ts:36]: please use https://angular.io/api/common/NgClass

**Review by klumba12 — CHANGES_REQUESTED** [review]

File: src/examples/grid-size/example-grid-size.component.html (hunk 1 above)
Comment (arcadiaEvgeniyCheremukhin) [reply src/examples/grid-size/example-grid-size.component.html]: Modified.

File: src/examples/grid-size/example-grid-size.component.ts (hunk 2 above)
Comment (arcadiaEvgeniyCheremukhin) [reply src/examples/grid-size/example-grid-size.component.ts:36]: Modified.

File: src/examples/grid-size/example-grid-size.component.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import { GridComponent } from 'ng2-qgrid';
+
+@Component({
+	selector: 'example-grid-size',
+	templateUrl: 'example-grid-size.component.html',
+	styleUrls: ['example-grid-size.component.scss'],
+	providers: [DataService],
+	changeDetection: ChangeDetectionStrategy.OnPush,
+	styles: [
+		`.plain{width:100%; height:100%;}
```
Comment (klumba12) [inline src/examples/grid-size/example-grid-size.component.ts:13]: move it please under scss file

Comment (klumba12) [reply src/examples/grid-size/example-grid-size.component.ts:13]: rename `plain` to `fill`

File: src/examples/grid-size/example-grid-size.component.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+		.small{width:500px; height:500px  !important;}`
+	]
+})
+export class ExampleGridSizeComponent {
+	static id = 'grid-size';
+
+	rows: Observable<Human[]>;
+	@ViewChild(GridComponent, { static: true }) grid: GridComponent;
+
+	gridClass = {
```
Comment (klumba12) [inline src/examples/grid-size/example-grid-size.component.ts:24]: rename gridClass to sizeClass and make it `string` like sizeClass = 'fill'

File: src/examples/grid-size/example-grid-size.component.ts [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+		plain: true,
+		medium: false,
+		small: false
+	}
+
+	constructor(dataService: DataService) {
+		this.rows = dataService.getPeople();
+	}
+
+	fillPlain() {
```
Comment (klumba12) [inline src/examples/grid-size/example-grid-size.component.ts:34]: just in template sizeClass = 'medium'

File: src/examples/grid-size/example-grid-size.component.html [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<button (click)="fillPlain()">100%</button>
```
Comment (klumba12) [inline src/examples/grid-size/example-grid-size.component.html:1]: sizeClass = 'fill'

File: src/examples/grid-size/example-grid-size.component.ts (hunk 3 above)
Comment (arcadiaEvgeniyCheremukhin) [reply src/examples/grid-size/example-grid-size.component.ts]: Modified

File: src/examples/grid-size/example-grid-size.component.ts (hunk 4 above)
Comment (arcadiaEvgeniyCheremukhin) [reply src/examples/grid-size/example-grid-size.component.ts]: Modified

File: src/examples/grid-size/example-grid-size.component.ts (hunk 5 above)
Comment (arcadiaEvgeniyCheremukhin) [reply src/examples/grid-size/example-grid-size.component.ts]: Modified

File: src/examples/grid-size/example-grid-size.component.html (hunk 6 above)
Comment (arcadiaEvgeniyCheremukhin) [reply src/examples/grid-size/example-grid-size.component.html]: Modified

**Review by klumba12 — APPROVED** [review]

**Pull request merged**